modules/monitoring/public/css/tables.less

```diff
@@ -209,8 +209,13 @@
 }
 
 // Wraps links, icons and meta in overviews
-.state-header {
+.state-header,
+.comment-author {
   .clearfix();
+
+  > a {
+    font-weight: bold;
```

Comment:
Is there room for a compromise? Maybe a less bold look?

Reply:
Since we use different OS fonts, the only way to ensure to have bold fonts is to use the descriptive value `bold`. If you use numeric values, some fonts behave different than others.